skins/pstat/dialog.css

```diff
@@ -285,7 +285,7 @@ div.cke_disabled .cke_dialog_ui_labeled_content div * {
     padding-right: 0
 }
 
-.cke_hc div.cke_dialog_ui_input_text, .cke_hc div.cke_dialog_ui_input_password, .cke_hc div.cke_dialog_ui_input_textarea, .cke_hc div.cke_dialog_ui_input_select, .cke_hc div.cke_dialog_ui_input_file {
+.cke_hc div.cke_dialog_ui_input_text, .cke_hc div.cke_dialog_ui_input_password, .cke_hc div.cke_dialog_ui_input_tel, .cke_hc div.cke_dialog_ui_input_textarea, .cke_hc div.cke_dialog_ui_input_select, .cke_hc div.cke_dialog_ui_input_file {
```

Comment:
all CSS updates follow the pattern: "Add `input.cke_dialog_ui_input_tel` and `div.cke_dialog_ui_input_tel` where `input.cke_dialog_ui_input_password` and  `div.cke_dialog_ui_input_password`"

As official styles do:

https://github.com/PolicyStat/ckeditor-dev/blob/5e6cc22e748766353317e31b12c8b7a452211da9/skins/kama/dialog_ie7.css#L77-L90